amgut/lib/data_access/sql_connection.py

```diff
@@ -1,72 +1,204 @@
-from __future__ import division
-
 # -----------------------------------------------------------------------------
-# Copyright (c) 2014--, The American Gut Development Team.
+# Copyright (c) 2014--, The Qiita Development Team.
```

Comment:
This code is copy pasted from qiita - I think it is ok leaving this copyright note.